dpnp/tests/third_party/cupy/random_tests/test_distributions.py

```diff
@@ -72,12 +72,14 @@ def test_beta(self, a_dtype, b_dtype):
             "shape": [(4, 3, 2), (3, 2)],
             "n_shape": [(), (3, 2)],
             "p_shape": [(), (3, 2)],
-            "dtype": _int_dtypes,  # to escape timeout
+            # "dtype": _int_dtypes,  # to escape timeout
+            "dtype": [None],  # no dtype supported
         }
     )
 )
 class TestDistributionsBinomial(RandomDistributionsTestCase):
 
+    @pytest.mark.skip()
```

Comment:
Was it intentional to skip test? Do we need to place a reason?

Reply:
Our implementation works only for scalars. This test passes zero-dimensional arrays which doesn't pass scalar check, as result these tests doesn't work at all.
I think we would return to these tests when would refactor random generators or whatever this test is.
I could place some reason though

Reply:
Added